app.js

```diff
@@ -0,0 +1,61 @@
+var AIchoice = ['rock', 'paper', 'scissors'];
+var rand = Math.floor(Math.random()*3);
```

Comment:
You only ever assign rand once - which means the computer picks at the beginning of the game and never changes its choice.